eng/scripts/get-aspire-cli.ps1

```diff
@@ -602,6 +602,101 @@ function Expand-AspireCliArchive {
     }
 }
 
+# Function to map quality to channel name
+function ConvertTo-ChannelName {
```

Comment:
@davidfowl Is this intentional?

Reply:
Yea the names are different unfortunately